packages/jsx/src/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { atom, Atom, AtomMaybe, CtxSpy, isAtom } from '@reatom/core'
+import { isObject} from '@reatom/utils'
+
+type ClassValue = AtomMaybe<
+  | string
+  | number
+  | boolean
+  | null
+  | undefined
+  | Array<ClassValue>
+  | Atom<ClassValue>
+  | Record<string, AtomMaybe<string | number | boolean | null | undefined | object>>
+  | ((ctx: CtxSpy) => ClassValue)
+>
+
+export const cn = (value: ClassValue): Atom<string> => atom((ctx) => reatomClassName(ctx, value))
+
+const reatomClassName = (ctx: CtxSpy, value: ClassValue): string => {
```

Comment:
A thing name starts with "reatom*" if it returns an atom or an action. Name it `parseClasses`, maube, or something like that